onadata/libs/utils/export_tools.py

```diff
@@ -864,4 +872,9 @@ def parse_request_export_options(params):
     if value_select_multiples and value_select_multiples in boolean_list:
         options['value_select_multiples'] = str_to_bool(value_select_multiples)
 
+    if params.get("repeat_index_tags") == DEFAULT_INDEX_TAGS:
```

Comment:
This is super confusing, it does seem like if a user specifies their set of tags then we will use the default ones instead of the user defined.
Was the intention to validate the user-supplied tags instead?